docs/integrate/universal-integrate-case/custom-marketplace-entrypoint.en.md

```diff
@@ -5,31 +5,48 @@ slug: custom-marketplace-entrypoint
 
-Currently we supports two mirror sites:
-
-1. China mirror site: [https://marketplace.smartide.cn](https://marketplace.smartide.cn)
-2. Official mirror site: [https://open-vsx.org](https://open-vsx.org)
 
 ## Configuration
 
-Generally, the initial project of the framework will use the configuration of the mirror site in two places, one is the `download script` to install the default plug-in, and the other is the `extension marketplace` configuration when the framework is started.
+Generally, the initial project of the framework will use the configuration of the mirror site in two places, one is the download script to install the default extensions, and the other is the extension marketplace configuration when the framework is started.
+
 
 ### How to modify the download script
 
-You just need to find files such as `scripts/download.js` and modify the site information used in it
+The download script (`scripts/download.js`) uses the Ant CloudIDE extension marketplace by default. If you need to switch, you can specify the extension marketplace type in the `download-extension` directive in the `package.json` file:
 
-```typescript
-- const api = 'https://open-vsx.org/api/';
-+ const api = 'https://marketplace.smartide.cn/api/'; // China Mirror
+```json
+{
+  "scripts": {
+    -"download-extension": "cross-env DEBUG=InstallExtension node scripts/download.js"
+    +"download-extension": "cross-env DEBUG=InstallExtension MARKETPLACE=openvsx node scripts/download.js"
+  }
+}
 ```
 
 ### How to modify the plugin market source
 
-You can refer to the code here: [node/start-server.ts#L18](https://github.com/opensumi/opensumi-module-samples/blob/main/example/src/node/start-server.ts# L18).
+You can refer to the code here: [node/start-server.ts#L18](https://github.com/opensumi/opensumi-module-samples/blob/main/example/src/node/start-server.ts# L18), add relevant parameter information in the configuration parameters of the Node process startup. The specific configuration is as follows:
 
-You only need to add the following parameter information to the configuration parameters of Node process startup:
+#### Ant CloudIDE Marketplace
+
+```typescript
+let opts: IServerAppOpts = {
+  ...
+  marketplace: {
+    endpoint: 'https://marketplace.opentrs.cn',
+    accountId: 'clcJKq_Gea47whxAJGrgoYqf',
+    masterKey: '_V_LPJ6Ar-1nrSVa05xDGBYp',
+  },
+  ...
+};                 
+```
+The accountId and masterKey used in the above configuration are the public secret key provided by the Ant CloudIDE Marketplace. This secret key can access all public extensions in the marketplace by default. If you need to customize the secret key, you can refer to the client management section in the [Ant CloudIDE Marketplace documentation](https://www.opentrs.cn/cloudide/documents/documentDetail?productStr=cloudide-20221026&nameSpace=trms2d/xyyfdt&slug=ooxr2vxp32r9hv4q). Through the custom secret key, users can access their private extensions hosted in the marketplace.
```

Comment:
`users can access their private extensions hosted in the marketplace.` -> `you can access your private extensions hosted in the marketplace.`